autoreload_inotify.c

```diff
@@ -14,9 +14,11 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with sxiv.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * nsxiv is distributed under the same conditions.
```

Comment:
Are we sure this is the proper way to deal with rename/rebranding?

Reply:
I suppose the GPLv2 license does not call for the preservation of the name, so we can just leave the top-most copyright notice for the author and convert the sxiv occurrences to nsxiv.